#
# old_revision [019c2c7a876819db59add22c98c7939e32d81161]
#
# patch "a"
#  from [03cfd743661f07975fa2f1220c5194cbaff48451]
#    to [023cc93272e5cbc11f128ffc90fbbe42f6f08d8c]
#
============================================================
--- a	03cfd743661f07975fa2f1220c5194cbaff48451
+++ a	023cc93272e5cbc11f128ffc90fbbe42f6f08d8c
@@ -1 +1 @@
-abc
+ls
